Contributing.md

```diff
@@ -82,4 +82,33 @@ to the TC and the TC uses its standard consensus seeking process to evaluate whe
 not to add this new member. Members who do not participate consistently at the level of
 a majority of the other members are expected to resign.
 
-
+# Project Captains
+
+The Express TC can designate captains for individual projects/repos in the
+three organizations. These captains are responsible for being the primary
+day-to-day maintainers of the repo on a technical and community front.
+They will ensure that decisions which effect the Express project at large are
+brought up with the TC (in meeting or over GitHub issues) but otherwise will
+have the full authority in the direction of the individual project.  These captains
+will be given full permissions on both the repo as well as the npm packages, and
+will follow the community guidelines for publishing and maintaining support.
+
+The Express TC can designate captains for individual projects/repos in the
+three organizations. These captains are responsible for being the primary
+day-to-day maintainers of the repo on a technical and community front.
+Repo captains are empowered with repo ownership and module publication rights.
+When there are conflicts, especially on topics that effect the Express project
+at large, captains are responsible to bring it up with the TC and drive
+those conflicts to resolution. Captains are also responsible in making sure
+community members follow the community guidelines, maintaining the repo
+and the registry, as well as in providing user support.
+
+## Current Project Captains
+
+- `path-to-regexp`: @blakeembrey
+- `multer`: @LinusU 
+- `expressjs.com`: @crandmck & @jonchurch
```

Comment:
I have a question on these: some like this one is not the actual _current_ captain. Shouldn't this be the list of the current ones and then we go through the process outlined here to add the new ones?

Reply:
Also curious why we only list some of the repos?

Reply:
(And they seem to be in an almost random ordering lol)

Reply:
@dougwilson -
 
> I have a question on these: some like this one is not the actual current captain. Shouldn't this be the list of the current ones and then we go through the process outlined here to add the new ones?

agreed, and removed the proposed names for future inclusion based on the process.

> Also curious why we only list some of the repos?

At this point, I don't know the full list, and I can't see a documentation on it. I specifically asked this in https://github.com/expressjs/discussions/issues/98#issuecomment-596481428 . Please let me know, so I shall update it.

> And they seem to be in an almost random ordering lol)

fixed, sorted those alphabetically.

PTAL!